Documentation/network/concepts/ipam/multi-pool.rst

```diff
@@ -239,7 +239,7 @@ Multi-Pool IPAM is a preview feature. The following limitations apply to Cilium
 Multi-Pool IPAM mode:
 
 .. warning::
-   - Transparent encryption is only supported with WireGuard and cannot be used with IPsec.
+   - Transparent encryption is supported with IPSec in tunnel routing mode or with WireGuard.
```

Comment:
nit: Since it is a limitation, maybe we should rephrase this as something like `IPSec is not supported in native routing mode. IPSec in tunnel mode and WireGuard (both in native routing and tunnel mode) are supported`.

Reply:
Yep, sounds more clear. I'll update it, thanks!